src/core/trulens/core/feedback/feedback.py

```diff
@@ -381,26 +298,20 @@ def _default_selectors(self):
 
         if len(par_names) == 1:
             # A single argument remaining. Assume it is record output.
-            selectors = {par_names[0]: mod_feedback_schema.Select.RecordOutput}
-            self._print_guessed_selector(
-                par_names[0], mod_feedback_schema.Select.RecordOutput
-            )
+            selectors = {par_names[0]: Select.RecordOutput}
+            self._print_guessed_selector(par_names[0], Select.RecordOutput)
 
             # TODO: replace with on_output ?
 
         elif len(par_names) == 2:
             # Two arguments remaining. Assume they are record input and output
             # respectively.
             selectors = {
```

Comment:
I love your refractor to remove `mod_*` the code is so much cleaner.

Reply:
There's quite a lot more cases across the repo so it would be more time consuming to fix but agree it helps with legibility.